### PR TITLE
Update crowdloans

### PR DESCRIPTION
**Pull request by stepanLav**
Adding new crowdloans:

- https://parachains.info/details/moonsama
- https://parachains.info/details/peaq
- https://parachains.info/details/invarch#about

### CODE REVIEW

File: crowdloan/polkadot-dev.json [hunk 1]
```diff
@@ -305,5 +305,31 @@
...
+    "paraid": "3334",
+    "name": "Moonsama",
+    "token": "SAMA",
+    "description": "Moonsama’s mission is to unlock the potential of Non Fungible Tokens (NFTs) as true elements of utility in the Web3 space.",
+    "website": "https://moonsama.com/?ref=parachains-info",
+    "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/Moonsama.svg"
+  },
+  {
+    "paraid": "3338",
+    "name": "peaq",
```
Comment (valentunn) [inline crowdloan/polkadot-dev.json:319]: Should it be lowercase?

File: crowdloan/polkadot-dev.json [hunk 2]
```diff
@@ -305,5 +305,31 @@
...
+    "token": "PEAQ",
+    "description": "peaq is the decentralized Web3 network powering the Economy of Things, designed to solve some of our biggest societal and economic challenges, it ensures that the incentives of people and increasingly intelligent machines are aligned.",
+    "website": "https://moonsama.com/",
```
Comment (valentunn) [inline crowdloan/polkadot-dev.json:322]: Wrong website

File: crowdloan/polkadot-dev.json (hunk 1 above)
Comment (valentunn) [reply crowdloan/polkadot-dev.json:319]: I see, on website it is the same

File: crowdloan/polkadot-dev.json (hunk 2 above)
Comment (stepanLav) [inline crowdloan/polkadot-dev.json:322]:
```suggestion
    "website": "https://www.peaq.network/",
```

**Review by leohar — APPROVED** [review]

**Pull request merged**